program/pimoroni

```diff
@@ -648,14 +648,15 @@ phat_menu() {
 
 TITLE="Pimoroni Product Installer"
 MAINLIST="\
-1/Enviro pHAT/\
-2/ESP IoT pHAT/\
-3/Explorer pHAT/\
-4/Micro Dot pHAT/\
-5/Mote pHAT/\
-6/pHAT DAC/\
-7/Scroll pHAT/\
-8/Unicorn pHAT"
+1/Automation pHAT/\
```

Comment:
this is a typical addition of a board. This is only ONE of the (minimum) TWO occurrences that needs to be added.

note that, if the list is to remain alphabetically ordered then the numbering needs to be amended for all following. It's a pretty crude system, but there is no real way around it within the simple (but robust) Zenity/Whiptail framework... menus are hardcoded for better or worst.